instagram/src/App.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import './App.css';
+import dummyData from './dummy-data';
+import SearchBar from './components/SearchBar/SearchBar.js';
+import PostListContainer from './components/PostListContainer/PostListContainer.js';
+import PostsPage from './components/PostContainer/PostsPage.js';
+
+
+class App extends Component {
+
+  constructor(){
+    super();
+    this.state = {
+      postData: []
+
+    }
+  }
+
+  componentDidMount(){
+    setTimeout(() => {
+      this.setState({postData: dummyData});
+
+    }, 500);
+
+  }
+
+  addComment = (commentObj, user) =>{
+    console.log(commentObj);
+    console.log(user);
+
+
+    let newPostData = this.state.postData;
+    console.log(newPostData);
+    for(let i=0; i < newPostData.length; i++){
```

Comment:
Really like this pattern. Make your changes, outside of setState, then on line #42, everything is nice and clean and easy to debug if something goes wrong 👍